File: 1662-check-if-two-string-arrays-are-equivalent/1662-check-if-two-string-arrays-are-equivalent.java
class Solution {
    public boolean arrayStringsAreEqual(String[] word1, String[] word2) {
        
        int i =0, j = 0;
        
        String newstr1 = "";
        String newstr2 = "";
        
        while(i < word1.length){
            newstr1 += word1[i];
            i ++;
        }
        
        
        while(j < word2.length){
            newstr2 += word2[j];
            j ++;
        }
    
        
        if(newstr1.equals(newstr2)){
            return true;
        }
            
        else{
            return false;
        }
        
    }
}

File: 1662-check-if-two-string-arrays-are-equivalent/1662-check-if-two-string-arrays-are-equivalent.py
class Solution:
    def arrayStringsAreEqual(self, word1: List[str], word2: List[str]) -> bool:
        #i, j = 0, 0
        #newstr1 = ""
        #newstr2 = ""
        #while i < len(word1):
        #    newstr1 += word1[i]
        #    i += 1
        
        #while j < len(word2):
        #   newstr2 += word2[j]
        #    j += 1
            
        #if newstr1 == newstr2:
        #    return True
        #else:
        #   return False
        
        str1 = "".join(word1)
        str2 = "".join(word2)
        if str1 == str2:
            return True
        else:
            return False
            
            
        
       